third_party/s2n-bignum/import.sh

```diff
@@ -49,7 +50,7 @@ mkdir ${TMP}
 
 echo "Fetching repository ..."
 git clone ${GITHUB_SERVER_URL}/${GITHUB_REPOSITORY} ${TMP} --branch ${GITHUB_TARGET} --single-branch > /dev/null
-GITHUB_COMMIT=$(cd ${TMP} >/dev/null; git rev-parse HEAD)
+GITHUB_COMMIT=$(cd ${TMP} > /dev/null; git checkout ${COMMIT_HASH} > /dev/null; git rev-parse HEAD)
```

Comment:
Suggestion: Follow the mlkem-native importer and only import the exact commit you care about. At the same time, remove distinction between GITHUB_TARGET and COMMIT_HASH:
```
git init >/dev/null
git remote add origin $GITHUB_SERVER_URL/$GITHUB_REPOSITORY >/dev/null
git fetch origin --depth 1 $GITHUB_SHA >/dev/null
git checkout FETCH_HEAD >/dev/null
```